BD-2230 Editing old Content Blocks with Link Aliasing enabled (#5684)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/templates/link_aliasing.md

New Content Blocks will have their links modified where Braze will append a `lid={{placeholder}}` to each link where applicable. This placeholder value is resolved when inserted into an email message variant.

> To modify the links within existing Content Blocks that were created before Braze enabled link aliasing, you'll have to duplicate the existing Content Blocks and modify the links within the duplicated Content Blocks.

Reviewer: ```suggestion
To modify the links within existing Content Blocks that were created before Braze enabled link aliasing, duplicate the existing Content Blocks, then modify the links within the duplicated Content Blocks.
```
